skadi-lib/build.gradle.kts

```
@@ -21,3 +20,73 @@ dependencies {
     testImplementation("com.google.truth:truth:${Dependencies.truth}")
     testImplementation("com.github.ologe:flow-test-observer:1.4.0")
 }
+
+val siteUrl = "https://github.com/Syex/skadi"
+val vcsUrl = "https://github.com/Syex/skadi.git"
+val libDescription = "A Kotlin JVM library featuring a redux-like architecture with coroutines"
+
+val libVersion = "0.1.0"
+
+// maven publish plugin
+val sourcesJar by tasks.registering(Jar::class) {
+    archiveClassifier.set("sources")
+    from(sourceSets.main.get().allSource)
+}
```

There is a nicer [helper function](https://docs.gradle.org/6.4/userguide/publishing_maven.html#publishing_maven:complete_example) for this:
```suggestion
java {
    withJavadocJar()
    withSourcesJar()
}
```